src/bindings/python/src/openvino/frontend/pytorch/ts_decoder.py

```diff
@@ -84,8 +85,24 @@ def _get_scripted_model(self, pt_module, example_inputs=None, skip_freeze=False)
             if example_inputs is None:
                 scripted = torch.jit.script(pt_module)
             else:
-                input_parameters, input_signature, pt_module, self._input_is_list = prepare_example_inputs_and_model(example_inputs, input_params, pt_module)
-                scripted = torch.jit.trace(pt_module, **input_parameters, strict=False)
+                input_parameters, input_signature, pt_module, self._input_is_list = prepare_example_inputs_and_model(
+                    example_inputs, input_params, pt_module)
+                gptq_patched = False
+
```

Comment:
I would recommend the following check before trying GPTQ patching:
`if getattr(model.config, "quantization_config", None) and model.config.quantization_config.quant_method == "gptq":`